pos_product_sync/static/src/js/pos_product_sync.js

```diff
@@ -18,11 +18,15 @@ odoo.define('pos_product_sync.pos', function (require) {
         on_barcode_updates: function(data){
             var self = this;
             if (data.message === 'update_product_fields' && data.product_ids && data.product_ids.length) {
```

Comment:
```suggestion
            var product_ids = data && data.product_ids && _.uniq(data.product_ids);
            if (product_ids && data.message === 'update_product_fields' && product_ids.length) {
```

Reply:
and update ``data.product_ids`` underneath ofcourse.

Reply:
I think, it's resolved now